server/src/main/java/com/google/musicanalysis/site/MusixRequest.java

```diff
@@ -0,0 +1,75 @@
+package com.google.musicanalysis.site;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/*
+    Handles gathering data from the Musixmatch API.
+    This is structured the way it is becuase I will be adding more
+    to each function.
+*/
+public class MusixRequest {
+
+  // Url building
+  private static final String BASE_URL = "https://api.musixmatch.com/ws/1.1/";
+  private String operation;
+  private static final String OUTPUT_REQS = "?format=jsonp&callback=callback&";
+  private String parameters;
+  private static final String apiKey = "&apikey=";
```

Comment:
nit: name this API_KEY since it's a const. Also, group all static variables together separately from your regular variables:

```
private String foo;
private String bar;

private static final ZOOP;
private static final DOOP;
```